### ISSUE TITLE
Support for Alternative Drive Types

### ISSUE THREAD

**Issue by Roba00** [wpilibsuite/PathWeaver#121]
@zengstellas Made this in my account by accident.

**Is your feature request related to a problem? Please describe.**  
Currently PathWeaver assumes your robot is a Tank Drive, but many teams use other drivetrains such as Mecanum and Swerve.

**Describe the solution you'd like**  
Have support for other drivetrains as stated above.

**Describe alternatives you've considered**  
We can take the waypoints from PathWeaver but not the Left-Right values for non-tank drives.

**Additional context**  
None.

**Comment by NicolasNewman** [comment]
I'd love to see this as well.

**Comment by jasondaming** [comment]
I agree this is a nice feature but with none of the teams I support doing Mec or Swerve I think it will be hard for me to implement this.  Would love to have outside help if interested.  Would be a great feature!

2021 is planned to have the ability to more easily import the waypoints into code so at least that should be helpful.
